components/Dialogs/examples/DialogsWithEmphasisButtonExampleViewController.swift

```diff
@@ -0,0 +1,86 @@
+// Copyright 2019-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+
+import MaterialComponents.MaterialButtons
+import MaterialComponents.MaterialDialogs
+import MaterialComponentsBeta.MaterialContainerScheme
+import MaterialComponentsBeta.MaterialDialogs_Theming
+
+class DialogsWithEmphasisButtonExampleViewController: UIViewController {
+
+  private let materialButton = MDCButton()
+  var containerScheme = MDCContainerScheme()
+
+  override func viewDidLoad() {
+    super.viewDidLoad()
+
+    view.backgroundColor = containerScheme.colorScheme?.backgroundColor ??
+      MDCSemanticColorScheme(defaults: .material201804).backgroundColor
+
+    materialButton.translatesAutoresizingMaskIntoConstraints = false
+    materialButton.setTitle("Material Alert With Emphasis Buttons", for: .normal)
+    materialButton.setTitleColor(UIColor(white: 0.1, alpha:1), for: .normal)
```

Comment:
Please use a Theming Extension for theming buttons.

Reply:
Removed this redundant line in 01b601d8b0da4ef34c91394e3e8f0b2b8ada6a50.
Theming extension was applied on https://github.com/material-components/material-components-ios/pull/6600/files/23f2461319868fb1bb4b998165e1101d9a4b751a#diff-0cbf49333bff15d9b5117827ee60fc63R38.